basic-sql-statements.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Basic SQL Statements
+toc: false
+---
+
+This page walks you through some of the most essential CockroachDB SQL statements. For a complete list of statements and their options, as well as details about data types and other concepts, see [SQL Reference](sql-reference.html).
+
+{{site.data.alerts.callout_info}}CockroachDB aims to provide standard SQL with extensions, but some standard SQL functionality is not yet available. Joins, for example, will be built into version 1.0. See our <a href="https://github.com/cockroachdb/cockroach/issues/2132">Product Roadmap</a> for more details.{{site.data.alerts.end}}   
+
+<img src="images/catrina_ramen.png" style="max-width: 200px;" />
+
+## Create a Database
+
+CockroachDB comes with a single default `system` database, which contains CockroachDB metadata and is read-only. To create a new database, use the `CREATE DATABASE` statement followed by a database name:
+
+```sql
+CREATE DATABASE db1;
+```
+
+Database names must follow [these rules](identifiers.html). To avoid an error in case the database already exists, you can include `IF NOT EXISTS`:
+
+```postgres
+CREATE DATABASE IF NOT EXISTS db1;
+```
+
+When you no longer need a database, use the `DROP DATABASE` statement followed by the database name to remove the database and all its objects:
+
+```sql
+DROP DATABASE db1;
+```
+
+## Show Databases
+
+To see all databases, use the `SHOW DATABASES` statement:
+
+```mysql
+SHOW DATABASES;
+```
+```
++----------+
+| Database |
++----------+
+| db1      |
+| system   |
++----------+
+```
+
+## Set the Default Database
+
+To set the default database, use the `SET DATABASE` statement:
+
+```sql
+SET DATABASE = db1;
+```
+
+When working with the default database, you don't need to reference it explicitly in statements. To see which database is currently the default, use the `SHOW DATABASE` statement (note the singular form):
+
+```sql
+SHOW DATABASE;
+```
+```
++----------+
+| DATABASE |
++----------+
+| db1      |
++----------+
+```
+
+## Create a Table
+
+To create a table, use the `CREATE TABLE` statement followed by a table name, the column names, the data type for each column, and the primary key constraint:
+
+```sql
+CREATE TABLE table1 (
+    column_a INT PRIMARY KEY,
+    column_b FLOAT,
+    column_c DATE,
+    column_d STRING
+);
+```
+
+Table and column names must follow [these rules](identifiers.html). Also, when you don't explicitly define a `PRIMARY KEY`, CockroachDB will automatically add a `rowid` column as the primary key.
+
+To avoid an error in case the table already exists, you can include `IF NOT EXISTS`:
+
+```sql
+CREATE TABLE IF NOT EXISTS table1 (
+    column_a INT PRIMARY KEY,
+    column_b FLOAT,
+    column_c DATE,
+    column_d STRING
+);
+```
+
+To verify that all columns were created correctly, use the `SHOW COLUMNS FROM` statement followed by the table name:
```

Comment:
This sounds a bit strange. If the table wasn't created correctly an error would have been returned. Perhaps reword the initial part as: `To show all of the columns from an existing table, use ...`